src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    public void handleOperation(FruitTransaction fruitTransaction, Storage storage) {
+        if (storage.getInventory().containsKey(fruitTransaction.getFruit())) {
+            storage.getInventory().compute(fruitTransaction.getFruit(),
+                    (k, currentQty) -> currentQty
+                                + fruitTransaction.getQuantity());
+        } else {
+            storage.getInventory().put(fruitTransaction.getFruit(),
```

Comment:
You can use `putIfAbsent` method to simplify the logic of adding a new fruit to the storage. This will also remove the need for the `if-else` block.